src/webui/webui.h

```diff
@@ -52,7 +54,12 @@ class WebUI final : public ApplicationComponent<QObject>
     Q_DISABLE_COPY_MOVE(WebUI)
 
 public:
-    explicit WebUI(IApplication *app);
+    enum TempPasswordOption : bool
```

Comment:
I couldn't find where it is used.

Reply:
> I couldn't find where it is used.

It is used in the place where WebUI is constructed:
https://github.com/qbittorrent/qBittorrent/blob/df893ef170b024974000a5d12e25696e849228c4/src/app/application.cpp#L893

Reply:
Shouldn't you use it here too:
```c++
explicit WebUI(IApplication *app, TempPasswordOption tempPasswordEnabled = DISABLE_TEMP_PASSWORD);
```

Reply:
I didn't intend to make a full-fledged type out of it, just a named constant to improve the readability of the code.

P.S. `tempPasswordEnabled = DISABLE_TEMP_PASSWORD` looks weird/ugly.

Reply:
This code will most likely gone when refactoring to avoid store plain text password.